GNUmakefile

```diff
@@ -50,16 +50,20 @@ cov:
 	open /tmp/coverage.html
 
 test: dev
-	go test -tags "$(GOTAGS)" -i ./...
-	go test -tags "$(GOTAGS)" -run '^$$' ./... > /dev/null
```

Comment:
I had added line 54 to force it to compile tests first before running them, to catch and fail fast if test compilation errors occur.  Any reason to remove that now? 
The -i flag only compiles dependencies of tests and not the tests themselves.